source/content/solr.md

```diff
@@ -46,10 +46,10 @@ For installation instructions and additional details, refer to [Enabling Solr fo
 
 - Solr can only be enabled by [Admin users or Users in Charge](/guides/account-mgmt/workspace-sites-teams/teams#site-level-roles-and-permissions).
 
-- Anything that takes more than 5 seconds to send to the Solr server to be indexed will timeout, which will block indexing. For example, large documents attached to Drupal nodes. In these cases, the developer must work with the content or code to exempt the nodes and/or files from being indexed. [Check Index and Batch sizes](#check-index-and-batch-sizes) for possible scenarios or more time-outs.
+- Anything that takes more than 5 seconds to send to the Solr server to be indexed will timeout, which will block indexing. For example, large documents are attached to Drupal nodes. In these cases, the developer must work with the content or code to exempt the nodes and/or files from being indexed. [Check Index and Batch sizes](#check-index-and-batch-sizes) for possible scenarios or more time-outs.
```

Comment:
```suggestion
- Anything that takes more than 5 seconds to send to the Solr server to be indexed will timeout, which will block indexing. For example, Drupal nodes with large document attached. In these cases, the developer must work with the content or code to exempt the nodes and/or files from being indexed. [Check Index and Batch sizes](#check-index-and-batch-sizes) for possible scenarios or more time-outs.
```

I think the idea here is that a Drupal node might not get indexed if there is a large document file attached to it.

Reply:
That's how this sentence read to me prior to the addition of "are" -- that nodes with large documents attached to them will block indexing.

Reply:
Agree With Brian's suggestion. Approved with those changes.